dockstore/dockstore-support
# Redeliver release events

**Description**
Redeliver release events. Code already delivers push and installation events. Did a little refactoring to avoid code duplication.

**Review Instructions**
1. Create a GitHub Release for a repo you have configured on Dockstore QA.
2. Ensure that you can see the event in app logs; copy the delivery id
3. In CloudShell, install Java, clone dockstore-support, build `./mvnw install -pl githubdelivery -am`
4. Follow instructions in README.md to redeliver event.

**Issue**
SEAB-6466

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install` in the project that you have modified (until https://ucsc-cgl.atlassian.net/browse/SEAB-5300 adds multi-module support properly)
- [x] If you are changing dependencies, check with dependabot to ensure you are not introducing new high/critical vulnerabilities
